feature-libs/checkout/b2b/components/checkout-delivery-address/checkout-delivery-address.component.html

```diff
@@ -1,15 +1,25 @@
 <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
   {{ 'checkoutAddress.shippingAddress' | cxTranslate }}
 </h2>
-<ng-container *ngIf="cards$ | async as cards; else loading">
-  <ng-container *ngIf="!shouldRedirect && !(isUpdated$ | async); else loading">
+
+<ng-container *ngIf="state$ | async as state">
+  <ng-container
+    *ngIf="!state.shouldRedirect && !state.isUpdating; else loading"
+  >
     <ng-container
-      *ngIf="cards?.length && !addressFormOpened; else newAddressForm"
+      *ngIf="
+        isAccountPayment || (state.cards?.length && !addressFormOpened);
+        then hasExistingAddresses;
```

Comment:
👍  only way that I can think of with the global state above to have the button 'outside' the spinner

Reply:
idea was to flatten all and just use state. place in other places, but I needed the back and continue button out when selecting the cards for both delivery address and payment method

Reply:
no need anymore since we're putting the buttons in the spinner too now